Makefile

```diff
@@ -0,0 +1,47 @@
+default: test
+
+test:
+	@go test -race ./lib/...
+fmt:
+	@go fmt ./lib/...
+license:
+	@go run license.go lib
+
+checkfmt:
+ifneq ($(shell gofmt -l lib),)
+	$(error code not fmted, run make fmt. $(shell gofmt -l lib))
+endif
+
+checklisence:
```

Comment:
Or even `check_license` to be consistent with the style of other tasks. Same with `check_fmt`.